README.adoc

```diff
@@ -162,7 +163,11 @@ annotated method, which is the total number of times the inventory is accessed.
 
 Apply the `@Gauge` annotation to the `getTotal()` method to track the number of systems that are stored in
 the inventory. Note that when the value of the gauge is retrieved, the underlying `getTotal()` method
-is called to return the size of the inventory.
+is called to return the size of the inventory. Note the additional metadata field:
+
+|===
+| `unit` | Use this field to scale the metric, if it is NONE, the metric name is used as is without appending the unit name and no scaling is applied. 
```

Comment:
I'm not sure I follow the description.  What does it mean to "scale the metric"?